testData/solutions/win32.json5

```diff
@@ -2021,6 +2021,137 @@
         ]
     },
 
+    "com.microsoft.windows.audioDescription": {
+        "name": "Windows video audio description feature",
+        "contexts": {
+            "OS": [
+                {
+                    "id": "win32",
+                    "version": ">=5.0"
+                }
+            ]
+        },
+        "settingsHandlers": {
+            "configure": {
+                "type": "gpii.windows.spiSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "getAction": "SPI_GETAUDIODESCRIPTION",
+                    "setAction": "SPI_SETAUDIODESCRIPTION",
+                    "uiParam": "struct_size",
+                    "pvParam": {
+                        "type": "struct",
+                        "name": "AUDIODESCRIPTION"
+                    },
+                    verifySettings: true
+                },
+                "supportedSettings": {
+                    "AudioDescriptionOn": {
+                        "schema": {
+                            "title": "Video audio description",
+                            "description": "Hear descriptions of what's happening in videos",
+                            "type": "boolean",
+                            "default": false
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "AudioDescriptionOn": {
+                        "transform": {
```

Comment:
Would `fluid.transforms.value` be enough? Passing `true` directly to `BOOL Enabled` would work.

Reply:
This have been changed due to recent conversations with @the-t-in-rtf .